data/tgbus

```diff
@@ -0,0 +1 @@
+tgbus.com
```

Comment:
dynamic advertisement detected:
```javascript
function (t, e, n) {
    'use strict';
    Object.defineProperty(e, '__esModule', {
      value: !0
    });
    var a = 'https://g1.tagtic.cn/v1/xingyou/c/',
    i = {
      methods: {
        generateScript: function (t, e) {
          var n = document.createElement('script'),
          a = '\n        SG_GG(' + t + ')\n      ',
          i = document.createTextNode(a);
          n.appendChild(i),
          document.getElementById(e).appendChild(n);
          var r = document.createElement('div');
          r.setAttribute('id', 'SG_GG_CONTAINER_' + t),
          document.getElementById(e).appendChild(r)
        }
      }
    },
    r = {
      methods: {
        generateCoupletScript: function (t) {
          var e = document.createElement('script');
          e.type = 'text/javascript',
          e.src = a + (t + '.js');
          var n = document.getElementsByTagName('script'),
          i = document.getElementsByTagName('script') [n.length - 1];
          i.parentNode.insertBefore(e, i)
        }
      }
    },
    s = {
      methods: {
        generateDropdownScript: function (t) {
          var e = document.createElement('script');
          e.type = 'text/javascript',
          e.src = a + (t + '.js');
          var n = document.getElementsByTagName('script'),
          i = document.getElementsByTagName('script') [n.length - 1];
          i.parentNode.insertBefore(e, i)
        }
      }
    };
    e.script = i,
    e.coupletScript = r,
    e.dropdownScript = s
  }
```

I suggest mark `g1.tagtic.cn` as advertisement.

Reply:
tracker detected in `http://bdtj.tagtic.cn/bi-sdk.1.2.1.js` referenced by `http://tgbus.cn`:
this will post user actions to `http://xy-log.tagtic.cn/mininfo/v1/logs/webtgbusweb`

I suggest mark `xy-log.tagtic.cn` as tracker.